composer.json

```
@@ -4,7 +4,7 @@
     "name": "mhsdesign/liveinspectorjsevents",
     "license": "GPL-3.0+",
     "require": {
-        "neos/neos-ui": "^5.3 || ^7.0 || ^8.0",
+        "neos/neos-ui": "^8.0",
         "mhsdesign/polyfillneosuiguestframeapi": "^1.0"
```

```suggestion
        "neos/neos-ui": "^8.0"
```

we can also get rid of that funky polyfill-package of mine :D

---

Done. Send it.